src/libse/SubtitleFormats/SubtitleFormat.cs

```diff
@@ -666,7 +667,7 @@ public static SubtitleFormat[] GetTextOtherFormats()
                 new DlDd(),
                 new Ted20(),
                 new Captionate(),
-                new TimeLineAscii(),
+                new TimeLineAscii(), // TODO: remove from here?
```

Comment:
Should we keep the format in `GetTextOtherFormats` since it now support writing to text? @niksedk